features/step_definitions/project_version_steps.rb

```diff
@@ -0,0 +1,16 @@
+Given /^project version (.+) exists for project (.+)(?: with creation date set (?:(1 day|(?:[2-9]|\d{2,}) days) ago))$/ do |name,project_name,nb_days|
```

Comment:
`(?:(1 day|(?:[2-9]|\d{2,}) days) ago)` what?

Replace with `(?:(\d+) days? ago)`, which also allows you to get rid of the ugly `split` at line 9.